storage/innobase/include/ut0ut.h

```diff
@@ -77,8 +77,8 @@ typedef time_t	ib_time_t;
 # define UT_RELAX_CPU() do { \
      volatile int32	volatile_var; \
      int32 oldval= 0; \
-     my_atomic_cas32(&volatile_var, &oldval, 1); \
-   } while (0)
+     my_atomic_cas32_strong_explicit(&volatile_var, &oldval, 1, MY_MEMORY_ORDER_RELAXED, MY_MEMORY_ORDER_RELAXED);\   
+} while (0)
```

Comment:
I'd rather keep this line as is or fix to use proper InnoDB coding style.
@dr-m, could you suggest the right indentation for define-s in InnoDB? Specifically for this UT_RELAX_CPU()?

Reply:
Is there a reason for `UT_RELAX_CPU()` to be a macro instead of an `inline` function in this special case? My proposal would be something like the following:
```c++
static inline void UT_RELAX_CPU()
{
	volatile int32 volatile_var;
	int32 oldval = 0;
	my_atomic_cas32_strong_explicit(
		&volatile_var, &oldval, 1,
		MY_MEMORY_ORDER_RELAXED,
		MY_MEMORY_ORDER_RELAXED);
}
```

Reply:
No reason. inline function is fine with me.

Reply:
Btw, with atomic operation ```volatile``` is not needed anymore.